orchestra/static/orchestra/team-info/team-info-card.directive.es6.js

```diff
@@ -77,6 +95,59 @@ export default function teamInfoCard (orchestraApi, helpers) {
         })
       }
 
+      teamInfoCard.handleKeydown = (taskId, assignmentId, stepSlug, worker) => {
+        if (!worker) {
+          teamInfoCard.assignTask(taskId, stepSlug)
+        } else {
+          teamInfoCard.reassignAssignment(taskId, assignmentId, stepSlug)
+        }
+      }
+
+      teamInfoCard.assignTask = (taskId, stepSlug) => {
+        teamInfoCard.loading = true
+        orchestraApi.assignTask(taskId, teamInfoCard.assignmentInput[stepSlug])
+          .then(function () {
+            delete teamInfoCard.assignmentInput[stepSlug]
+            teamInfoCard.loadTeamInfo()
+          }, function (response) {
+            teamInfoCard.assignmentInput[stepSlug] = ''
+            var errorMessage = 'Error assigning task.'
+            if (response.status === 400) {
+              errorMessage = response.data.message
+            }
+            window.alert(errorMessage)
+          })
+          .finally(function () {
+            teamInfoCard.loading = false
+          })
+      }
+
+      teamInfoCard.reassignAssignment = (taskId, assignmentId, stepSlug) => {
+        teamInfoCard.loading = true
+        orchestraApi.reassignAssignment(assignmentId, teamInfoCard.assignmentInput[stepSlug])
+          .then(function () {
+            delete teamInfoCard.assignmentInput[stepSlug]
+          }, function (response) {
+            var errorMessage = 'Error reassigning worker.'
+            if (response.status === 400) {
+              errorMessage = response.data.message
+            }
+            window.alert(errorMessage)
+          })
+          .finally(function () {
+            teamInfoCard.loadTeamInfo()
+            teamInfoCard.loading = false
+          })
+      }
+
+      teamInfoCard.toggleShowUnassigned = () => {
+        teamInfoCard.showUnassigned = !teamInfoCard.showUnassigned
+      }
+
+      teamInfoCard.showReassignmentInput = (stepSlug) => {
```

Comment:
Are we using `teamInfoCard.showReassignmentInput ` and `teamInfoCard.displayReassignmentInput` somewhere?

Reply:
Removed :)